pkgs/development/compilers/rust/1_39_0.nix

```diff
@@ -1,3 +1,8 @@
+# After updating check ...
+# 1. if rustc can produce rust binaries on x86_64-linux, aarch64-linux and x86_64-darwin:
+#    i.e. nix build -f . fd or @GrahamcOfBorg build fd on github
+# 2. if our LLVM version matches with rust upstream
+# 3. if firefox or thunderbird still build on x86_64-linux
```

Comment:
What are the implications of this? E.g. if firefox fails to build, should we simply refrain from updating Rust?

Reply:
Yes. Because this is a channel blocker. A way to fix it in that case is to to pin rust to an older version for firefox. However breaking firefox in unstable is undesired.